FirebaseDatabase/Sources/Api/FIRDatabaseQuery.m

```diff
@@ -95,32 +95,36 @@ - (void)validateQueryEndpointsForParams:(FQueryParams *)params {
             if (params.indexStartKey != [FUtilities minName] &&
                 params.indexStartKey != [FUtilities maxName]) {
                 [NSException raise:INVALID_QUERY_PARAM_ERROR
-                            format:@"Can't use queryStartingAtValue:childKey: "
+                            format:@"Can't use queryStartingAtValue:childKey:, "
+                                   @"queryStartingAfterValue:childKey:, "
                                    @"or queryEqualTo:andChildKey: in "
                                    @"combination with queryOrderedByKey"];
             }
             if (![params.indexStartValue.val isKindOfClass:[NSString class]]) {
                 [NSException
                      raise:INVALID_QUERY_PARAM_ERROR
-                    format:
-                        @"Can't use queryStartingAtValue: with other types "
-                        @"than string in combination with queryOrderedByKey"];
+                    format:@"Can't use queryStartingAtValue: or "
+                           @"queryStartingAfterValue: "
+                           @"with other types than string in combination with "
```

Comment:
Might be easier to understand if it said "with non-string types when used with queryOrderedByKey:"